apps/api_web/lib/api_web/controllers/live_facility_controller.ex

```diff
@@ -35,6 +37,21 @@ defmodule ApiWeb.LiveFacilityController do
     response(429, "Too Many Requests", Schema.ref(:TooManyRequests))
   end
 
+  defp ensure_path_matches_version(conn, _) do
```

Comment:
Because you know that you came from one of these paths (because of the router), you should only need to check one of them.

Reply:
Fixed, but it seems less readable to me like that.